app/src/main/res/values/strings_main.xml

```diff
@@ -66,14 +66,14 @@
     <string name="dialog_title_last_used">Last used</string>
     <string name="dialog_title_keystore_error">KeyStore error</string>
 
-    <string name="dialog_msg_auth">Please enter your device credentials to start andOTP.</string>
+    <string name="dialog_msg_auth">Please enter your device credentials in order to start andOTP.</string>
     <string name="dialog_msg_confirm_delete">Are you sure you want do remove the account \"%1$s\"?</string>
 
-    <string name="dialog_msg_last_used">In order for andOTP to recognize which token was used last
-        you have to have \"tap to reveal\" enabled or use the copy button.\n\nThis message will not
-        be shown again.</string>
+    <string name="dialog_msg_last_used">In order for andOTP to recognize which token was last used,
+        you will have to enable <b>\"tap to reveal\"</b> feature in the <b>Settings</b> page, 
```

Comment:
We should add a `the` before `<b>\"tap to reveal\"</b> feature`

Reply:
Ok. Point noted. Would effect the chances ASAP